metadata.rb

```diff
@@ -296,6 +300,14 @@
           :description => "ODirect",
           :type => 'string'
 
+attribute "ndb/SpinMethod",
+          :description => "SpinMethod",
+          :type => 'string'
+
+attribute "ndb/TotalMemoryConfig",
+          :description => "TotalMemoryConfig",
```

Comment:
as rondb docs are not up yet it would be nice to have couple of words here. I dont know what the default value of -1  means. I guess it takes the entire system memory.

Reply:
Are these values defined in "percentage"

Reply:
I added a more detailed description